### ISSUE TITLE
2-node hacluster and kubeapi load balancers

### ISSUE THREAD

**Issue by aieri** [canonical-web-and-design/ubuntu.com#5709]
Page https://ubuntu.com/kubernetes/docs/hacluster contains a couple of imprecisions:
* hacluster does not require a minimum of 3 units, it can work just fine with 2, provided you set cluster_count to 2. Split brains will be an issue, but the cluster will work
* the ha-cluster-vips option is actually spelled in the singular ha-cluster-vip. It is however correct that multiple space-separated addresses are supported.

---
*Reported from: https://ubuntu.com/kubernetes/docs/hacluster*

**Comment by solazio** [comment]
Hi @aieri thank you for reporting this. We will fix this issue as early as possible.

**Comment by pmahnke** [comment]
@evilnick what do you want to do with this bug?

**Comment by anthonydillon** [comment]
@pmahnke @evilnick can you discuss this process of these issues when you meet again please?

**Comment by evilnick** [comment]
Hi, sorry for the delay in getting to this.

* The VIP(s) issue has already been addressed in docs in some previous work
* While technically true that HACluster will work with two nodes, that is not the recommended. however, I take the point that it is imprecise, so I will reword it to imply only that the suggested usage is 3 or more nodes.

@pmahnke @anthonydillon - I think if there are more issues like this which are explicitly about docs they will get more visibility on https://github.com/charmed-kubernetes/kubernetes-docs/. In future I will just migrate them to that repo

**Comment by evilnick** [comment]
closed in favour of above
